.github/workflows/check-conventional-commits.yml

```diff
@@ -116,3 +116,60 @@ jobs:
           header: tip-conventional-commits
           hide: true
           hide_classify: 'RESOLVED'
+
+  label-pr:
+    needs: lint
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
This did not seem to take effect 🤔 @lachieh do I need to also specify `contents: read` here?

Reply:
I believe that it will not run because it is not yet merged. As in, it will only run this job on new PRs after this one is merged.

Reply:
Oh, you might be able to do a test PR against your fork+branch?

Reply:
Ok, yeah. Had a look, I had the same issue on #3911 and then pushed the branch into the org and was able to run correctly in #3917. Essentially, you are not allowed allowed to just create a PR from your own fork and have it perform actions on the main repo via API.